libs/application/templates/financial-aid/src/fields/AboutForm/AboutForm.tsx

```diff
@@ -24,8 +22,9 @@ const AboutForm = () => {
       </Box>
 
       <PrivacyPolicyAccordion
-        municipalityPageName="reykjavik.is"
-        municipalityPageUrl="https://reykjavik.is/"
+        municipalityPageUrl={
+          application.externalData.nationalRegistry?.data?.municipality.homepage
```

Comment:
missing question mark in front homepage, I think that is optional so it can be undefined 👀